Dikubot/Webapp/Data/Emojis/EmojiService.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Dikubot.Extensions.EmojiSelector;
+using Newtonsoft.Json;
+using System.IO;
+using System.Reflection;
+using System.Text;
+using Dikubot.Webapp.Shared.Emojis;
+using Microsoft.AspNetCore.Components;
+using Dikubot.Webapp.Data.Emojis;
+using Syncfusion.Lic.util.encoders;
+
+namespace Dikubot.Webapp.Data
+{
+    public class EmojiService
+    {
+        
+        public List<EmojiGroup> EmojiLists;
+        private Dictionary<string, string> _twitterEmojis;
+        
+        public string emptyEmoji = @"<svg xmlns=""http://www.w3.org/2000/svg"" viewBox=""0 0 36 36""></svg>";
```

Comment:
I don't assume this variable changes, so it should be immutable. I think it is called final or const in C#.